onnxscript/tools/benchmark/export_model.py

```diff
@@ -0,0 +1,157 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
```

Comment:
Reminder to use the new license header

Reply:
Ok, maybe that's worth checking as well.